dubbo-rpc/dubbo-rpc-xml/pom.xml

```diff
@@ -0,0 +1,170 @@
+<!--
+    - Copyright 1999-2011 Alibaba Group.
+    -
+    - Licensed under the Apache License, Version 2.0 (the "License");
+    - you may not use this file except in compliance with the License.
+    - You may obtain a copy of the License at
+    -
+    - http://www.apache.org/licenses/LICENSE-2.0
+    -
+    - Unless required by applicable law or agreed to in writing, software
+    - distributed under the License is distributed on an "AS IS" BASIS,
+    - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    - See the License for the specific language governing permissions and
+    - limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.dubbo</groupId>
+        <artifactId>dubbo-rpc</artifactId>
+        <version>2.7.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>dubbo-rpc-xml</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+    <description>The xml rpc module of dubbo project</description>
+    <properties>
+        <jetty_version>9.3.24.v20180605</jetty_version>
+        <mortbay_jetty_version>6.1.26</mortbay_jetty_version>
+        <xmlrpc_version>3.1.3</xmlrpc_version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <skip_maven_deploy>false</skip_maven_deploy>
+    </properties>
+    <url>https://github.com/apache/incubator-dubbo</url>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+            <comments>A business-friendly OSS license</comments>
+        </license>
+    </licenses>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.portlet</groupId>
```

Comment:
The external dependencies should go into dubbo-dependency-bom.

Reply:
Agree with raft0131

Reply:
I moved to dubbo-dependency-bom everything that isn't an org.apache dependency